recipes/glshaderpp/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} example.cpp)
+target_link_libraries(${PROJECT_NAME} CONAN_PKG::glshaderpp CONAN_PKG::glew)
```

```suggestion

find_package(glshaderpp CONFIG REQUIRED)

add_executable(${PROJECT_NAME} example.cpp)
target_link_libraries(${PROJECT_NAME} glshaderpp::glshaderpp glew::glew)
```

Avoid CONAN_PKG. It will be deprecated on Conan 2.0